var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -33,7 +33,8 @@ class Dyninst(Package):
     url = "https://github.com/dyninst/dyninst/archive/v9.2.0.tar.gz"
     list_url = "http://www.dyninst.org/downloads/dyninst-8.x"
 
-    version('9.3.2', git="https://github.com/dyninst/dyninst.git", tag='v9.3.2')
+    version('10.0.alpha', git="https://github.com/dyninst/dyninst.git", branch='master')
```

Comment:
I would just call this version `master` or `develop`. `master` is a moving target, if you give it a version number now that version number will rapidly become incorrect. If you do this, `preferred=True` will no longer be necessary below.

Reply:
@adamjstewart Yes, I will do that.  I thought about that too.  I have some changes ready for that and will check them in when I figure out the libelf - Mac issue.